### ISSUE TITLE
receive script failed icefilms  1.07 and 1.14 XBMC4XBOX v30962

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ankurp/anarchintosh-projects#41]
```
What steps will reproduce the problem?
1. try to load icefilms addon
2.
3.

What is the expected output? What do you see instead?
expect icefilms addon to load, receive script failed error instead

What version of the product are you using? On what operating system?
XBMC4XBOX v30962, icefilms 1.01 addon loads and can see movie list however when 
clicked on, actual video stream not showing, have since tried to update to 
version icefilms 1.07 and now to the latest icefilms version 1.0.14 both 
version 1.07 and 1.14 gives script failed error.

Please provide any additional information below.
To sum it up icefilms 1.01 worked perfectly a few months ago, now only shows 
movie names but not the actual files to load video, (perhaps a change on 
icefilms servers) icefilms  1.07 and 1.14 fails to load and give “script 
failed” error

I do hope something can be done about this indeed,


```

Original issue reported on code.google.com by `cid...@hotmail.com` on 17 Oct 2011 at 6:18
